src/rust/catnip/runtime/mod.rs

```diff
@@ -254,31 +253,28 @@ impl DPDKRuntime {
 
         println!("dev_info: {:?}", dev_info);
         let mut port_conf: rte_eth_conf = unsafe { MaybeUninit::zeroed().assume_init() };
-        port_conf.rxmode.max_rx_pkt_len = if use_jumbo_frames {
+        port_conf.rxmode.max_lro_pkt_size = if use_jumbo_frames {
```

Comment:
Why are we using a different field now? Or has it just been renamed?

Reply:
They changed some variables and constants.